pkg/daemon/ceph/client/info.go

```diff
@@ -63,6 +63,8 @@ type ClusterInfo struct {
 type MonInfo struct {
 	Name     string `json:"name"`
 	Endpoint string `json:"endpoint"`
+	// Whether detected out of quorum by rook. May be different from actual ceph quorum.
+	OutOfQuorum bool `json:"outOfQuorum"`
```

Comment:
What if the mon is just started it will be also skipped that mon?

Can we just query the mon that is in the config map and remove the duplicate ones??

Reply:
> What if the mon is just started it will be also skipped that mon?

Mons that are just being started are added to the config. They are only marked out of quorum when Ceph says they are out of quorum when rook runs the health check. So the mons are innocent until proven guilty.

> Can we just query the mon that is in the config map and remove the duplicate ones??

This was the approach I originally started with, but then realized only the out-of-quorum duplicate mon should be removed from the config. If we pick the wrong one, the config will be even less healthy.